tests/Form/InputColorTest.php

```
@@ -0,0 +1,983 @@
+            "Failed asserting that element renders correctly with 'value' attribute.",
+        );
+    }
+
+    public function testThrowInvalidArgumentExceptionForSettingColorspace(): void
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
+            Message::VALUE_NOT_IN_LIST->getMessage(
+                'invalid-value',
+                'colorspace',
+                implode("', '", Enum::normalizeArray(Colorspace::cases())),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'testThrowInvalidArgumentExceptionForSettingColorspace'. (undefined)

(StaticAccess)